recipes-kernel/kernel-tests/kernel-performance-tests-files/test_kernel_cyclictest_fio.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+source ./run-cyclictest
+
+# start background disk I/O load
+mkdir -p .fio
+fio fio.cfg --ioengine="sync" --runtime="$TEST_DURATION" --ramp_time=1m --output="results/fio-sync-`date +'%Y_%m_%d-%H_%M_%S'`.log" > /dev/null &
```

Comment:
Same comment about having a more absolute location to store the `.fio` directory.

Reply:
/tmp is not going to have enough space for this. The fio benchmark creates large files during the test. The folder is removed at the end of the test so it shouldn't matter but I can look for some other location in /var or something.